atomic

```diff
@@ -213,6 +213,37 @@ if __name__ == '__main__':
     imagesp.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
 
+    # atomic migrate
+    migratep = subparser.add_parser("migrate", help=_("migrate docker from one backend storage "
+                                                    "to another"))
+    migrate_subparser = migratep.add_subparsers(help=_("migrate commands"))
+
+   # atomic export
+    exportp = migrate_subparser.add_parser(
+        "export", help=_("export a docker instance into a filesystem directory"),
+        epilog="export a docker instance. "
+        "The export command would export docker images, "
+        "containers and volumes into a filesystem directory.")
+    exportp.set_defaults(func=atomic.Export)
+    exportp.add_argument("--graph", dest="graph", default="/var/lib/docker",
+                        help=_("Root of the Docker runtime (Default: /var/lib/docker)"))
+    exportp.add_argument("--dir", dest="export_location",
+                         default="/var/lib/docker-migrate",
+                         help=_("Path for exporting docker (Default: /var/lib/docker-migrate)"))
+	
```

Comment:
Not consistant on Docker versus docker, probably better to just use containers.

Should migrate to /var/lib/atomic not /var/lib/docker-migrate.